.env.example

```diff
@@ -1,5 +1,4 @@
-MQTT_BROKER = "test.mosquitto.org"
-MQTT_BROKER_PORT = "1883"
-MQTT_INPUT_TOPIC_PREFIX="BRE/calculateWinterSupplementInput/"
-MQTT_OUTPUT_TOPIC_PREFIX="BRE/calculateWinterSupplementOutput/"
-MQTT_TOPIC_ID="TOPIC ID"
```

Comment:
Removed `MQTT_TOPIC_ID` from `.env.example` as it is no longer needed to process requests. Refer to README for more details.